mesheryctl/internal/cli/root/system/config.go

```diff
@@ -49,7 +50,7 @@ type k8sContext struct {
 func getContexts(configFile, tokenPath string) ([]string, error) {
 	client := &http.Client{}
 
-	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	mctlCfg, err := config2.GetMesheryCtl(viper.GetViper())
```

Comment:
Why `config2`? I think there is no other package or variable with the name `config` - @anirudhjain75

Reply:
There is one inside system.go. Do have a look you'll see

Reply:
yes but why would that matter here?